README.md

```diff
@@ -1,2 +1,111 @@
+The gene counts were derived from quasi-mapping thousands of SRA fastq files against both reference transcriptomes.
+While each compendia contains gene counts for thousands of samples, 545 samples were identified as PA14 strain while 861 samples were identified as PAO1 strain (documented [here](https://github.com/greenelab/core-accessory-interactome/blob/master/data/metadata/SRA_annotations.tsv)). 
+
+During compendia construction, Doing et al. noted that known gene-gene relationships could be recovered using correlation analysis of gene counts.
+As depicted in the image below, genes that are part of the same operon and thus presumably co-transcribed on polycistronic transcripts have gene counts with high correlation coefficients after normalization.
+
+![](https://i.imgur.com/jHkwLn6.png)
+
+This repository tests whether a similar pattern can be recovered using FracMinHash abundances.
+We initially focus on the PA14 compendium as it is smaller. 
+
+## Things tested
+
+1. Sub sampled k-mer abundance distributions. Do these distributions look similar to gene count distributions (negative binomial)? 
+    + [20220322_compare_kmer_and_gene_count_dists.ipynb](notebooks/20220322_compare_kmer_and_gene_count_dists.ipynb)
+2. K-mer coverage for genes and operons. Do most genes and most operons have at least one hash (k-mer) in the FracMinHash sketch?
+    + [20220325_hashes_in_transcripts_and_operons.ipynb](notebooks/20220325_hashes_in_transcripts_and_operons.ipynb)
+3. Normalization techniques and correlation analysis. Can we recover co-expression patterns using correlation analysis of normalized k-mer abundances?
+    + DESeq2 median of ratios normalization: [20220328_correlations_mrnorm.ipynb](notebooks/20220328_correlations_mrnorm.ipynb)
+    + DESeq2 variance stabilized transformation: [20220330_correlations_vstnorm.ipynb](notebooks/20220330_correlations_vstnorm.ipynb)
+    + Seurat log normalization: [20220330_correlations_lognorm.ipynb](notebooks/20220330_correlations_lognorm.ipynb)
+    + Seurat centered log ratio normalization: [20220330_correlations_clrnorm.ipynb](notebooks/20220330_correlations_clrnorm.ipynb)
+    + Seurat counts per million normalization: [20220330_correlations_rcnorm.ipynb](notebooks/20220330_correlations_rcnorm.ipynb) 
+
+## Results and conclusions
+
+**Results**
+
+See notebooks for more details.
+
++ K-mer abundance distributions are weird
++ Many genes don't have a k-mer, but most operons do
++ Many spurious correlations remain after normalization, obscuring signal from intra-operon correlations. Median of ratios and centered log ratio normalization appear to have performed the best, however many spurious correlations remain after normalization. In situations where we won't have gene annotations available, this would make it difficult to determine which k-mers are actually part of the same operon and which are not. 
```

Comment:
```suggestion
+ Many spurious correlations (e.g. high correlation coefficients for k-mers that are not in the same operons) remain after normalization, obscuring signal from intra-operon correlations. Median of ratios and centered log ratio normalization appear to have performed the best, however many spurious correlations remain after normalization. In situations where we won't have gene annotations available, this would make it difficult to determine which k-mers are actually part of the same operon and which are not. 
```